src/libraries/Microsoft.Extensions.Primitives/ref/Microsoft.Extensions.Primitives.cs

```diff
@@ -167,6 +167,11 @@ public partial struct Enumerator : System.Collections.Generic.IEnumerator<Micros
         public static bool operator ==(string?[]? left, Microsoft.Extensions.Primitives.StringValues right) { throw null; }
         public static implicit operator string?(Microsoft.Extensions.Primitives.StringValues values) { throw null; }
         public static implicit operator string?[]?(Microsoft.Extensions.Primitives.StringValues value) { throw null; }
+#if NET9_0_OR_GREATER
+#pragma warning disable CS3006 // Overloaded method 'StringValues.implicit operator ReadOnlySpan<string?>(in StringValues)' differing only in ref or out, or in array rank, is not CLS-compliant.
+        public static implicit operator System.ReadOnlySpan<string?> (in Microsoft.Extensions.Primitives.StringValues value) { throw null; }
+#pragma warning restore CS3006
+#endif
```

Comment:
We generally don't use ifdefs in ref files. You'll want to create a separate Microsoft.Extensions.Primitives.netcoreapp.cs next to this one with just the new API, e.g. https://github.com/dotnet/runtime/blob/main/src/libraries/System.IO.Hashing/ref/System.IO.Hashing.netcoreapp.cs